packages/server/providers.yaml

```diff
@@ -9,10 +9,13 @@ asana:
     auth_mode: OAUTH2
     authorization_url: https://app.asana.com/-/oauth_authorize
     token_url: https://app.asana.com/-/oauth_token
+    refresh_url: https://app.asana.com/-/oauth_token
```

Comment:
We should omit the `refresh_url` if it's the same as the `token_url`. 

I think in general we should keep each template entry as small as possible, omitting optional params, because it reduces the potential for error and makes templates less intimidating.

Reply:
Awesome.

Reply:
This has been done.